### PR TITLE
correct (new) link

### PR DESCRIPTION
**Pull request by ahmadrasyidsalim**

### CODE REVIEW

File: locale/en/docs/meta/topics/dependencies.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ There are several dependencies that Node.js relies on to work the way it does.
 The V8 library provides Node.js with a JavaScript engine, which Node.js
 controls via the V8 C++ API. V8 is maintained by Google, for use in Chrome.
 
-* [Documentation](https://v8docs.nodesource.com/)
+* [Documentation](https://v8.dev/)
```
Comment (Trott) [inline locale/en/docs/meta/topics/dependencies.md]: Would https://v8.dev/docs be better since it's labeled "Documentation"?

Comment (ahmadrasyidsalim) [reply locale/en/docs/meta/topics/dependencies.md]: I think it will be better, I'll change it again.

**Review by nschonni — APPROVED** [review]
LGTM. Do you know if there are i18n docs on that site as well? Took a quick look but didn't see anything that looked like a language switcher

File: locale/en/docs/meta/topics/dependencies.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ There are several dependencies that Node.js relies on to work the way it does.
 The V8 library provides Node.js with a JavaScript engine, which Node.js
 controls via the V8 C++ API. V8 is maintained by Google, for use in Chrome.
 
-* [Documentation](https://v8docs.nodesource.com/)
+* [Documentation](https://v8.dev/docs)
```
Comment (richardlau) [inline locale/en/docs/meta/topics/dependencies.md:29]: Just an observation but the old URL and new URL are pointing at different things. The old URL was a link to V8 API documentation, which is useful to people embedding V8 (e.g. Node.js core developers) or writing addons (although `N-API` or using `nan` would be preferable to directly using the V8 API's to protect addons from breaking when the V8 API changes). 

The new URL doesn't contain any of this API documentation, although there is an issue requesting it (https://github.com/v8/v8.dev/issues/234).

Comment (MaledongGit) [reply locale/en/docs/meta/topics/dependencies.md]: @richardlau: This was fixed ;)

**Review by ghost — APPROVED** [review]

File: locale/en/docs/meta/topics/dependencies.md (hunk 2 above)
Comment (richardlau) [reply locale/en/docs/meta/topics/dependencies.md]: > 
> 
> @richardlau: This was fixed ;)

I see no difference from when I maded the observation -- the contents of the new link are different to the contents of the old link.

**Review by aymen94 — APPROVED** [review]

**Review by Aissaoui-Ahmed — APPROVED** [review]

**Review by MylesBorins — APPROVED** [review]
LGTM

**Pull request merged**